test-cases/artifacts/debian9.yaml

```diff
@@ -0,0 +1,18 @@
+cluster_backend: 'gce'
+gce_image: 'https://www.googleapis.com/compute/v1/projects/centos-cloud/global/images/family/debian-9'
+gce_instance_type_db: 'n1-standard-2'
```

Comment:
isn't all those are the default ? i.e. `gce_instance_type_db` and `gce_root_disk_size_db`  and `gce_root_disk_type_db` `gce_n_local_ssd_disk_db` ?
I think we don't need to repeat it all.

Reply:
Per discussion with Bentsi we decided to have these parameters defined here explicitly. And we can use smaller instances just for packaging verification.

Reply:
o.k.